src/NameRegistry.sol

```diff
@@ -884,28 +908,41 @@ contract NameRegistry is
     //////////////////////////////////////////////////////////////*/
 
     /**
-     * @notice Move the fname from the current owner to the pool and renew it for another year.
+     * @notice Move the fnames froms the current owners to desired destinations and renew them for another week.
```

Comment:
this line has a typo and an inaccuracy, i believe it should be: 

`Move the fnames from to their new destinations and renews them for a year if they expire within the next 30 days`

Reply:
@varunsrin Are we supposed to renew them for another year? Because currently, the reclaim in the main branch renews them for the `RENEWAL_PERIOD` which is 30 days.
 
I think this might have been an error so I will update my code accordingly!

Reply:
lets keep the existing renew logic (30 days is fine) and just make sure the comment matches it correctly.